pkg/runtime/from_config.go

```diff
@@ -21,12 +21,14 @@ import (
 var (
 	regCrd  *regexp.Regexp
 	regURL  *regexp.Regexp
+	regGRPC *regexp.Regexp
 	regFile *regexp.Regexp
 )
 
 func init() {
 	regCrd = regexp.MustCompile("^core.openfeature.dev/")
 	regURL = regexp.MustCompile("^https?://")
+	regGRPC = regexp.MustCompile("^" + grpc.Prefix)
```

Comment:
Do we need to handle `grpc` as well as `grpcs`? How would we indicate that the remote server expects the connection to be encrypted?

Reply:
Good point. 

As I [highlighted](https://github.com/open-feature/flagd/pull/297#discussion_r1101958011), my current approach considers a configuration option to disable tls connectivity. If this is not set, then default is to dial with system certs or if overriding cert is provided, use that for tls connectivity. 

You can see the draft PR highlights here - https://github.com/open-feature/flagd/pull/398/files#diff-5bd4f361280cd88498f36d821f08f603ba13af72ad0c1ce0313d7b3fa1ed5af5R185-R218

Reply:
I will resolve this conversation in this context. We can handle this with TLS related PR coming up soon - https://github.com/open-feature/flagd/pull/398